### ISSUE TITLE
Closed issues in source code

### ISSUE THREAD

**Issue by pyfisch** [servo/servo#19685]
These issues are mentioned in the source code as FIXMEs/TODOs etc. but are currently closed.

* #228
* #2003
* #2012
* #2260
* #2261
* #2262
* #4283
* #4311
* #9604
* #12749
* #14213
* #15569
* servo/webrender#28

Search with `rg "\((.*?#\d+.*?)\)" -trust -r '$1' -o -N --no-filename`.

Probably either the comment is outdated or the issue should not be closed.

**Comment by tigercosmos** [comment]
#15569 is opened again.

**Comment by CYBAI** [comment]
It seems all of mentioned issues are closed 👀

**Comment by pyfisch** [comment]
@CYBAI Yeah, that's why they should probably (after checking) be removed from the source code.

**Comment by atouchet** [comment]
The reference to #12749 was removed in #26407.

**Comment by lumiscosity** [comment]
These are now all gone except for:
- 4283\. This should probably just be replaced with a link to [#perform-a-microtask-checkpoint](https://html.spec.whatwg.org/multipage/webappapis.html#perform-a-microtask-checkpoint) or removed entirely? The issue doesn't seem to have any useful info for this at a glance.
- 9604\. I don't think this is actually done? The number is still hard-coded, at least. Maybe this is just done somewhere up the calls, I haven't checked it that thoroughly...
